### ISSUE TITLE
Exisiting RabbitMQ vhost and user causes error when running 'pupet apply' multiple times

### ISSUE THREAD

**Issue by kaazoo** [puppetlabs/puppetlabs-openstack#115]
```puppet
Info: Class[Rabbitmq::Config]: Scheduling refresh of Class[Rabbitmq::Service]
Info: Class[Rabbitmq::Service]: Scheduling refresh of Service[rabbitmq-server]
Notice: /Stage[main]/Rabbitmq::Service/Service[rabbitmq-server]: Triggered 'refresh' from 1 events
Error: Execution of '/usr/sbin/rabbitmqctl add_vhost /' returned 2: Creating vhost "/" ...
Error: vhost_already_exists: /

Error: /Stage[main]/Nova::Rabbitmq/Rabbitmq_vhost[/]/ensure: change from absent to present failed: Execution of '/usr/sbin/rabbitmqctl add_vhost /' returned 2: Creating vhost "/" ...
Error: vhost_already_exists: /

Error: Execution of '/usr/sbin/rabbitmqctl add_user openstack testing' returned 2: Creating user "openstack" ...
Error: user_already_exists: openstack

Error: /Stage[main]/Nova::Rabbitmq/Rabbitmq_user[openstack]/ensure: change from absent to present failed: Execution of '/usr/sbin/rabbitmqctl add_user openstack testing' returned 2: Creating user "openstack" ...
Error: user_already_exists: openstack
```

When I want to run 'puppet apply' again, I have to delete the vhost and user:
```
sudo rabbitmqctl delete_vhost /
sudo rabbitmqctl delete_user openstack
```

**Comment by cmurphy** [comment]
Yes, this is an issue with the rabbitmq module. The fix was released as 5.0.0. This module should be compatible with that version when #133 is merged, but we won't be able to update the dependency in the metadata because the stackforge modules are still depending on an old version. However you can still install the newer version manually.

**Comment by cmurphy** [comment]
Closing since this is inactionable for now, but the workaround is to install with the Puppetfile or manually upgrade your rabbitmq module if using the puppet module install tool.
